app/views/programs/students/_student_list.html.erb

```diff
@@ -46,6 +47,13 @@
             <td class="mi_tbody_td">
               <%= student.meeting_with_admin_date %>
             </td>
+            <td class="mi_tbody_td">
+              <% if student.notes.count > 0 %>
```

Comment:
_student.notes.count_ is again repeated on line 53. Consider storing it in a variable and then use it at both places.

Reply:
Where do you suggest to put this variable? Create a method in the application_helper?
Will it make any difference if instead of 'if student.notes.count > 0' I write 'f student.notes.present?'?

Reply:
Consider defining an instance variable in controller and assign it with _student.notes.count_. Then, you can use this variable in the view.

Reply:
This is the index page. The controller's index method has @students. There is no student there.

Reply:
Before line 51, try by adding _<% notesCount = student.notes.count %>_. Then, you can use variable _notesCount_ on line 51 and 53.

Reply:
I don't like doing that in the view file.
I changed the if statement from 'if student.notes.count > 0' to 'if student.notes.present?'
Now there is no repetition.